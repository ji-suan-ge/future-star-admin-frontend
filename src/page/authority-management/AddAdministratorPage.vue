<template>
  <el-container>
    <el-header>
      <el-page-header @back="goBack" content="添加管理员"></el-page-header>
    </el-header>
    <el-main>
      <el-table
        :data="administrators"
        stripe>
        <el-table-column
          fixed
          prop="name"
          label="姓名"
          :width="infoWidth">
          <template slot-scope="scope">
            <el-input placeholder="请输入姓名" v-model="scope.row.name"></el-input>
          </template>
        </el-table-column>
        <el-table-column
          prop="account"
          label="账号"
          :width="infoWidth">
          <template slot-scope="scope">
            <el-input placeholder="请输入账号" v-model="scope.row.account"></el-input>
          </template>
        </el-table-column>
        <el-table-column
          prop="password"
          label="密码"
          :width="infoWidth">
          <template slot-scope="scope">
            <el-input placeholder="请输入密码" v-model="scope.row.password" show-password></el-input>
          </template>
        </el-table-column>
        <el-table-column
          label="招生管理权限"
          align="center"
          :width="permissionWidth">
          <template slot-scope="scope">
            <el-switch
              v-model="scope.row.privilege.enrollment"
              :active-color="permissionActiveColor">
            </el-switch>
          </template>
        </el-table-column>
        <el-table-column
          label="课程管理权限"
          align="center"
          :width="permissionWidth">
          <template slot-scope="scope">
            <el-switch
              v-model="scope.row.privilege.semester"
              :active-color="permissionActiveColor">
            </el-switch>
          </template>
        </el-table-column>
        <el-table-column
          label="校友活动管理权限"
          align="center"
          :width="permissionWidth">
          <template slot-scope="scope">
            <el-switch
              v-model="scope.row.privilege.activity"
              :active-color="permissionActiveColor">
            </el-switch>
          </template>
        </el-table-column>
        <el-table-column
          label="校友资料管理权限"
          align="center"
          :width="permissionWidth">
          <template slot-scope="scope">
            <el-switch
              v-model="scope.row.privilege.student"
              :active-color="permissionActiveColor">
            </el-switch>
          </template>
        </el-table-column>
        <el-table-column
          label="操作"
          align="center"
          :width="operationWidth">
          <template slot-scope="scope">
            <el-button type="primary" size="small" @click="addAdministrator(scope.row)" round>提交</el-button>
            <el-button type="plain" size="small" @click="clearText(scope.row)" round>清空</el-button>
          </template>
        </el-table-column>
      </el-table>
    </el-main>
    <el-footer>
      <el-row type="flex" justify="center">
        <el-button type="primary" size="medium" @click="addAll()" round>一键提交</el-button>
        <el-button type="plain" size="medium" @click="clearTextAll()" round>一键清空</el-button>
      </el-row>
      <el-row type="flex" justify="center">
        <el-button type="primary" size="medium" @click="insertInfoArea()" round>继续添加</el-button>
      </el-row>
    </el-footer>
  </el-container>
</template>

<script>
export default {
  name: 'AddAdministrator',
  data () {
    return {
      permissionWidth: 180,
      infoWidth: 160,
      operationWidth: 180,
      permissionActiveColor: '#13ce66',
      administrators: [
        {
          name: '',
          account: '',
          password: '',
          privilege: {
            enrollment: 2,
            semester: 2,
            activity: 2,
            student: 2
          }
        }
      ]
    }
  },
  created () {
    for (let i = 0; i < this.administrators.length; i++) {
      this.administrators[i].privilege.enrollment = this.administrators[i].privilege.enrollment === 1
      this.administrators[i].privilege.semester = this.administrators[i].privilege.semester === 1
      this.administrators[i].privilege.activity = this.administrators[i].privilege.activity === 1
      this.administrators[i].privilege.student = this.administrators[i].privilege.student === 1
    }
  },
  methods: {
    goBack: function () {
      this.$router.go(-1)
    },
    addAdministrator (administrator) {
      console.log(administrator)
      administrator.privilege.enrollment = administrator.privilege.enrollment ? 1 : 2
      administrator.privilege.semester = administrator.privilege.semester ? 1 : 2
      administrator.privilege.activity = administrator.privilege.activity ? 1 : 2
      administrator.privilege.student = administrator.privilege.student ? 1 : 2
      let that = this
      that.axios.post('/administrator/administrator', {
        name: administrator.name,
        account: administrator.account,
        password: administrator.password,
        privilege: {
          enrollment: administrator.privilege.enrollment,
          semester: administrator.privilege.semester,
          activity: administrator.privilege.activity,
          student: administrator.privilege.student
        }
      }).then(function (response) {
        if (response.data.code === '2000') {
          administrator = {
            name: '',
            account: '',
            password: '',
            privilege: {
              enrollment: 2,
              semester: 2,
              activity: 2,
              student: 2
            }
          }
          that.$message({
            type: 'success',
            message: '添加成功',
            duration: 2000
          })
        } else {
          that.$message({
            type: 'error',
            message: '请求出错',
            duration: 2000
          })
        }
      }).catch(function (error) {
        console.log(error)
        that.$message({
          type: 'error',
          message: '服务器内部错误',
          duration: 2000
        })
      })
    },
    clearText: function (administrator) {
      administrator.name = ''
      administrator.account = ''
      administrator.password = ''
      administrator.privilege.enrollment = false
      administrator.privilege.semester = false
      administrator.privilege.activity = false
      administrator.privilege.student = false
    },
    clearTextAll: function () {
      this.administrators = [{
        name: '',
        account: '',
        password: '',
        privilege: {
          enrollment: 2,
          semester: 2,
          activity: 2,
          student: 2
        }
      }]
    },
    insertInfoArea: function () {
      this.administrators.push({
        name: '',
        account: '',
        password: '',
        privilege: {
          enrollment: 2,
          semester: 2,
          activity: 2,
          student: 2
        }
      })
    },
    addAll: function () {

    }
  }
}
</script>

<style scoped>

</style>
