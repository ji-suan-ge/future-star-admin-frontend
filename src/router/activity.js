import CreateActivityPage from '../page/CreateActivityPage'
import ShowActivityParticipantsPage from '../page/ShowActivityParticipantsPage'
import EditExistingActivityPage from '../page/EditExistingActivityPage'
import ShowActivityInformationPage from '../page/ShowActivityInformationPage'

export default [
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  },
  {
    path: '/show-participants',
    name: 'ShowActivityParticipantsPage',
    component: ShowActivityParticipantsPage
  },
  {
    path: '/edit-activity',
    name: 'EditExistingActivityPage',
    component: EditExistingActivityPage
  },
  {
    path: '/show-activity-details',
    name: 'ShowActivityInformationPage',
    component: ShowActivityInformationPage
  }
]
