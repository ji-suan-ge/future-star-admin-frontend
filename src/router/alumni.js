import ShowAlumniInformationPage from '../page/alumni-management/ShowAlumniInformationPage'
import EditAlumniInformationPage from '../page/alumni-management/EditAlumniInformationPage'
import ShowAlumniEditHistoryPage from '../page/alumni-management/ShowAlumniEditHistoryPage'
import ShowAlumniHistoryInformationPage from '../page/alumni-management/ShowAlumniHistoryInformationPage'

export default [
  {
    path: '/show-all-information',
    name: 'ShowAlumniInformationPage',
    component: ShowAlumniInformationPage
  },
  {
    path: '/edit-alumni-information',
    name: 'EditAlumniInformationPage',
    component: EditAlumniInformationPage
  },
  {
    path: '/show-edit-history',
    name: 'ShowAlumniEditHistoryPage',
    component: ShowAlumniEditHistoryPage
  },
  {
    path: '/show-history-information',
    name: 'ShowAlumniHistoryInformationPage',
    component: ShowAlumniHistoryInformationPage
  }
]
