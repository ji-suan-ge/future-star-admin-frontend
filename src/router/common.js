import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import homeChildren from './home-children'

export default [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      ...homeChildren
    ]
  }
]
