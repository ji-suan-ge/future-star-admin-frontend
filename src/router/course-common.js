import DetailOfSemesterPage from '../page/course-common/DetailOfSemesterPage'
import DetailOfClassPage from '../page/course-common/DetailOfClassPage'
import DetailOfCoursePage from '../page/course-common/DetailOfCoursePage'
import DetailOfTeacherPage from '../page/course-common/DetailOfTeacherPage'
import DetailOfSubCoursePage from '../page/course-common/DetailOfSubCoursePage'
import EditSemesterInfoPage from '../page/course-common/EditSemesterInfoPage'
import EditClassInfoPage from '../page/course-common/EditClassInfoPage'
import EditCourseInfoPage from '../page/course-common/EditCourseInfoPage'
import EditTeacherInfoPage from '../page/course-common/EditTeacherInfoPage'
import AddCoursePage from '../page/course-common/AddCoursePage'
import AddCourseResourcePage from '../page/course-common/AddCourseResourcePage'
import StudentInClass from '../page/course-common/StudentInClass'

export default [
  {
    path: '/semester-detail',
    name: 'DetailOfSemesterPage',
    component: DetailOfSemesterPage
  },
  {
    path: '/class-detail',
    name: 'DetailOfClassPage',
    component: DetailOfClassPage
  },
  {
    path: '/course-detail',
    name: 'DetailOfCoursePage',
    component: DetailOfCoursePage
  },
  {
    path: '/teacher-detail',
    name: 'DetailOfTeacherPage',
    component: DetailOfTeacherPage
  },
  {
    path: '/sub-course-detail',
    name: 'DetailOfSubCoursePage',
    component: DetailOfSubCoursePage
  },
  {
    path: '/edit-semester-info',
    name: 'EditSemesterInfoPage',
    component: EditSemesterInfoPage
  },
  {
    path: '/edit-class-info',
    name: 'EditClassInfoPage',
    component: EditClassInfoPage
  },
  {
    path: '/edit-course-info',
    name: 'EditCourseInfoPage',
    component: EditCourseInfoPage
  },
  {
    path: '/edit-teacher-info',
    name: 'EditTeacherInfoPage',
    component: EditTeacherInfoPage
  },
  {
    path: '/add-course',
    name: 'AddCoursePage',
    component: AddCoursePage
  },
  {
    path: '/add-course-resource',
    name: 'AddCourseResourcePage',
    component: AddCourseResourcePage
  },
  {
    path: '/student-in-class',
    name: 'StudentInClass',
    component: StudentInClass
  }
]
