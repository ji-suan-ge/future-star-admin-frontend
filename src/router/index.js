import Vue from 'vue'
import VueRouter from 'vue-router'
import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import EnrollmentManagementPage from '../page/EnrollmentManagementPage'
import CourseManagementPage from '../page/CourseManagementPage'
import ActivityManagementPage from '../page/ActivityManagementPage'
import AlumniManagementPage from '../page/AlumniManagementPage'
import AuthorityManagementPage from '../page/AuthorityManagementPage'
import CreateActivityPage from '../page/CreateActivityPage'
import AddAdministrator from '../page/AddAdministrator'
import WelcomePage from '../page/WelcomPage'

Vue.use(VueRouter)

const routes = [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      {
        path: 'welcome',
        name: 'WelcomePage',
        component: WelcomePage
      },
      {
        path: 'enrollment-management',
        name: 'EnrollmentManagementPage',
        component: EnrollmentManagementPage
      },
      {
        path: 'course-management',
        name: 'CourseManagementPage',
        component: CourseManagementPage
      },
      {
        path: 'activity-management',
        name: 'ActivityManagementPage',
        component: ActivityManagementPage
      },
      {
        path: 'alumni-management',
        name: 'AlumniManagementPage',
        component: AlumniManagementPage
      },
      {
        path: 'authority-management',
        name: 'AuthorityManagementPage',
        component: AuthorityManagementPage
      },
      {
        path: 'add-administrator',
        name: 'AddAdministrator',
        component: AddAdministrator
      }
    ]
  },
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  }
]

const router = new VueRouter({
  routes
})

export default router
