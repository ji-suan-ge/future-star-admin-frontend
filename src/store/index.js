import Vue from 'vue'
import Vuex from 'vuex'

Vue.use(Vuex)

export default new Vuex.Store({
  state: {
    activeIndexOfNavigation: '0',
    activeNameOfCourse: 'first',
    activeNameOfEnrollment: 'first',
    activeNameOfActivity: 'first',
    activeSemesterOfCourse: '',
    semesters: '',
    currentSemester: '',
    classes: '',
    currentClass: '',
    courses: '',
    currentCourse: '',
    infoOfAnAlumni: null
  },
  mutations: {
    changeActiveIndexOfNavigation (state, activeIndex) {
      state.activeIndexOfNavigation = activeIndex
    },
    changeActiveNameOfEnrollment (state, activeName) {
      state.activeNameOfEnrollment = activeName
    },
    changeActiveNameOfCourse (state, activeName) {
      state.activeNameOfCourse = activeName
    },
    changeActiveNameOfActivity (state, activeName) {
      state.activeNameOfActivity = activeName
    },
    changeActiveSemesterOfCourse (state, activeID) {
      state.activeSemesterOfCourse = activeID
    },
    changeSemesters (state, semesters) {
      state.semesters = semesters
    },
    changeInfoOfAnAlumni (state, alumniInfo) {
      state.infoOfAnAlumni = alumniInfo
    },
    changeCurrentSemester (state, semester) {
      state.currentSemester = semester
    },
    changeClasses (state, classes) {
      state.classes = classes
    },
    changeCurrentClass (state, currentClass) {
      state.currentClass = currentClass
    },
    changeCourses (state, courses) {
      state.courses = courses
    },
    changeCurrentCourse (state, currentCourse) {
      state.currentCourse = currentCourse
    },
    initializeStateWhileLogin (state) {
      state.activeIndexOfNavigation = '0'
      state.activeNameOfCourse = 'first'
      state.activeNameOfEnrollment = 'first'
      state.activeNameOfActivity = 'first'
      state.activeSemesterOfCourse = ''
      state.semesters = ''
      state.currentSemester = ''
      state.classes = ''
      state.currentClass = ''
      state.courses = ''
      state.currentCourse = ''
      state.infoOfAnAlumni = null
    },
    initializeStateWhileSwitch (state) {
      state.activeNameOfCourse = 'first'
      state.activeNameOfEnrollment = 'first'
      state.activeNameOfActivity = 'first'
      state.activeSemesterOfCourse = ''
      state.semesters = ''
      state.currentSemester = ''
      state.classes = ''
      state.currentClass = ''
      state.courses = ''
      state.currentCourse = ''
      state.infoOfAnAlumni = null
    }
  },
  actions: {
    changeActiveIndexOfNavigation (context, activeIndex) {
      context.commit('changeActiveIndexOfNavigation', activeIndex)
    },
    changeActiveNameOfEnrollment (context, activeName) {
      context.commit('changeActiveNameOfEnrollment', activeName)
    },
    changeActiveNameOfCourse (context, activeName) {
      context.commit('changeActiveNameOfCourse', activeName)
    },
    changeActiveNameOfActivity (context, activeName) {
      context.commit('changeActiveNameOfActivity', activeName)
    },
    changeActiveSemesterOfCourse (context, activeID) {
      context.commit('changeActiveSemesterOfCourse', activeID)
    },
    changeSemesters (context, semesters) {
      context.commit('changeSemesters', semesters)
    },
    changeInfoOfAnAlumni (context, alumniInfo) {
      context.commit('changeInfoOfAnAlumni', alumniInfo)
    },
    changeCurrentSemester (context, semester) {
      context.commit('changeCurrentSemester', semester)
    },
    changeClasses (context, classes) {
      context.commit('changeClasses', classes)
    },
    changeCurrentClass (context, currentClass) {
      context.commit('changeCurrentClass', currentClass)
    },
    changeCourses (context, courses) {
      context.commit('changeCourses', courses)
    },
    changeCurrentCourse (context, currentCourse) {
      context.commit('changeCurrentCourse', currentCourse)
    },
    initializeStateWhileLogin (context) {
      context.commit('initializeStateWhileLogin')
    },
    initializeStateWhileSwitch (context) {
      context.commit('initializeStateWhileSwitch')
    }
  },
  getters: {
    getActiveIndexOfNavigation (state) {
      return state.activeIndexOfNavigation
    },
    getActiveNameOfEnrollment (state) {
      return state.activeNameOfEnrollment
    },
    getActiveNameOfCourse (state) {
      return state.activeNameOfCourse
    },
    getActiveNameOfActivity (state) {
      return state.activeNameOfActivity
    },
    getActiveSemesterOfCourse (state) {
      return state.activeSemesterOfCourse
    },
    getSemesters (state) {
      return state.semesters
    },
    getInfoOfAnAlumni (state) {
      return state.infoOfAnAlumni
    },
    getCurrentSemester (state) {
      return state.currentSemester
    },
    getClasses (state) {
      return state.classes
    },
    getCurrentClass (state) {
      return state.currentClass
    },
    getCourses (state) {
      return state.courses
    },
    getCurrentCourse (state) {
      return state.currentCourse
    }
  },
  modules: {
  }
})
