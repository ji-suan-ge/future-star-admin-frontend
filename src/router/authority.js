import AddAdministratorPage from '../page/authority-management/AddAdministratorPage'

export default [
  {
    path: '/add-administrator',
    name: 'AddAdministratorPage',
    component: AddAdministratorPage
  }
]
