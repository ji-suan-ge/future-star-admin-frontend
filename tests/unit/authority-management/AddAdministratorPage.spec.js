import { createLocalVue, mount } from '@vue/test-utils'
import AddAdministratorPage from '../../../src/page/authority-management/AddAdministratorPage'
import Vuex from 'vuex'
import ElementUI from 'element-ui'
import VueRouter from 'vue-router'
import store from '../../../src/store'
import router from '../../../src/router'

describe('AddAdministratorPage', () => {
  const localVue = createLocalVue()
  localVue.use(Vuex)
  localVue.use(ElementUI)
  localVue.use(VueRouter)
  const wrapper = mount(AddAdministratorPage, {
    store,
    router,
    localVue
  })
  test('快照测试', async () => {
    await expect(wrapper.html()).toMatchSnapshot()
  })
  test('是一个 Vue 实例', async () => {
    await expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
