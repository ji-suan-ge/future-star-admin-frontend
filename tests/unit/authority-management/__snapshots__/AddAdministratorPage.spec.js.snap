// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddAdministratorPage 快照测试 1`] = `
<section class="el-container is-vertical">
  <header class="el-header" style="height: 60px;">
    <div class="el-page-header">
      <div class="el-page-header__left"><i class="el-icon-back"></i>
        <div class="el-page-header__title">返回</div>
      </div>
      <div class="el-page-header__content">添加管理员</div>
    </div>
  </header>
  <main class="el-main">
    <div class="el-table el-table--fit el-table--striped el-table--scrollable-x el-table--enable-row-transition">
      <div class="hidden-columns">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
      <div class="el-table__header-wrapper">
        <table cellspacing="0" cellpadding="0" border="0" class="el-table__header" style="width: 1380px;">
          <colgroup>
            <col name="el-table_1_column_1" width="160">
            <col name="el-table_1_column_2" width="160">
            <col name="el-table_1_column_3" width="160">
            <col name="el-table_1_column_4" width="180">
            <col name="el-table_1_column_5" width="180">
            <col name="el-table_1_column_6" width="180">
            <col name="el-table_1_column_7" width="180">
            <col name="el-table_1_column_8" width="180">
          </colgroup>
          <thead class="">
            <tr class="">
              <th colspan="1" rowspan="1" class="el-table_1_column_1     is-hidden is-leaf">
                <div class="cell">姓名</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_2     is-leaf">
                <div class="cell">账号</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_3     is-leaf">
                <div class="cell">密码</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_4  is-center   is-leaf">
                <div class="cell">招生管理权限</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_5  is-center   is-leaf">
                <div class="cell">课程管理权限</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_6  is-center   is-leaf">
                <div class="cell">校友活动管理权限</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_7  is-center   is-leaf">
                <div class="cell">校友资料管理权限</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_8  is-center   is-leaf">
                <div class="cell">操作</div>
              </th>
            </tr>
          </thead>
        </table>
      </div>
      <div class="el-table__body-wrapper is-scrolling-left">
        <table cellspacing="0" cellpadding="0" border="0" class="el-table__body" style="width: 1380px;">
          <colgroup>
            <col name="el-table_1_column_1" width="160">
            <col name="el-table_1_column_2" width="160">
            <col name="el-table_1_column_3" width="160">
            <col name="el-table_1_column_4" width="180">
            <col name="el-table_1_column_5" width="180">
            <col name="el-table_1_column_6" width="180">
            <col name="el-table_1_column_7" width="180">
            <col name="el-table_1_column_8" width="180">
          </colgroup>
          <tbody>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1   is-hidden">
                <div class="cell">
                  <div class="el-input">
                    <!----><input type="text" autocomplete="off" placeholder="请输入姓名" class="el-input__inner">
                    <!---->
                    <!---->
                    <!---->
                    <!---->
                  </div>
                </div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2  ">
                <div class="cell">
                  <div class="el-input">
                    <!----><input type="text" autocomplete="off" placeholder="请输入账号" class="el-input__inner">
                    <!---->
                    <!---->
                    <!---->
                    <!---->
                  </div>
                </div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3  ">
                <div class="cell">
                  <div class="el-input el-input--suffix">
                    <!----><input type="password" autocomplete="off" placeholder="请输入密码" class="el-input__inner">
                    <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><i class="el-input__icon el-icon-view el-input__clear"></i><!----></span>
                    <!----></span>
                    <!---->
                    <!---->
                  </div>
                </div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">
                  <div role="switch" aria-checked="true" class="el-switch is-checked"><input type="checkbox" name="" true-value="true" class="el-switch__input">
                    <!----><span class="el-switch__core" style="width: 40px; border-color: #13ce66; background-color: rgb(19, 206, 102);"></span>
                    <!---->
                  </div>
                </div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">
                  <div role="switch" aria-checked="true" class="el-switch is-checked"><input type="checkbox" name="" true-value="true" class="el-switch__input">
                    <!----><span class="el-switch__core" style="width: 40px; border-color: #13ce66; background-color: rgb(19, 206, 102);"></span>
                    <!---->
                  </div>
                </div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">
                  <div role="switch" aria-checked="true" class="el-switch is-checked"><input type="checkbox" name="" true-value="true" class="el-switch__input">
                    <!----><span class="el-switch__core" style="width: 40px; border-color: #13ce66; background-color: rgb(19, 206, 102);"></span>
                    <!---->
                  </div>
                </div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">
                  <div role="switch" aria-checked="true" class="el-switch is-checked"><input type="checkbox" name="" true-value="true" class="el-switch__input">
                    <!----><span class="el-switch__core" style="width: 40px; border-color: #13ce66; background-color: rgb(19, 206, 102);"></span>
                    <!---->
                  </div>
                </div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-round">
                    <!---->
                    <!----><span>提交</span></button> <button type="button" class="el-button el-button--plain el-button--small is-round">
                    <!---->
                    <!----><span>清空</span></button></div>
              </td>
            </tr>
            <!---->
          </tbody>
        </table>
        <!---->
        <!---->
      </div>
      <!---->
      <div class="el-table__fixed" style="width: 160px;">
        <div class="el-table__fixed-header-wrapper">
          <table cellspacing="0" cellpadding="0" border="0" class="el-table__header" style="width: 1380px;">
            <colgroup>
              <col name="el-table_1_column_1" width="160">
              <col name="el-table_1_column_2" width="160">
              <col name="el-table_1_column_3" width="160">
              <col name="el-table_1_column_4" width="180">
              <col name="el-table_1_column_5" width="180">
              <col name="el-table_1_column_6" width="180">
              <col name="el-table_1_column_7" width="180">
              <col name="el-table_1_column_8" width="180">
            </colgroup>
            <thead class="">
              <tr class="">
                <th colspan="1" rowspan="1" class="el-table_1_column_1     is-leaf">
                  <div class="cell">姓名</div>
                </th>
                <th colspan="1" rowspan="1" class="el-table_1_column_2     is-hidden is-leaf">
                  <div class="cell">账号</div>
                </th>
                <th colspan="1" rowspan="1" class="el-table_1_column_3     is-hidden is-leaf">
                  <div class="cell">密码</div>
                </th>
                <th colspan="1" rowspan="1" class="el-table_1_column_4  is-center   is-hidden is-leaf">
                  <div class="cell">招生管理权限</div>
                </th>
                <th colspan="1" rowspan="1" class="el-table_1_column_5  is-center   is-hidden is-leaf">
                  <div class="cell">课程管理权限</div>
                </th>
                <th colspan="1" rowspan="1" class="el-table_1_column_6  is-center   is-hidden is-leaf">
                  <div class="cell">校友活动管理权限</div>
                </th>
                <th colspan="1" rowspan="1" class="el-table_1_column_7  is-center   is-hidden is-leaf">
                  <div class="cell">校友资料管理权限</div>
                </th>
                <th colspan="1" rowspan="1" class="el-table_1_column_8  is-center   is-hidden is-leaf">
                  <div class="cell">操作</div>
                </th>
              </tr>
            </thead>
          </table>
        </div>
        <div class="el-table__fixed-body-wrapper" style="top: 0px;">
          <table cellspacing="0" cellpadding="0" border="0" class="el-table__body" style="width: 1380px;">
            <colgroup>
              <col name="el-table_1_column_1" width="160">
              <col name="el-table_1_column_2" width="160">
              <col name="el-table_1_column_3" width="160">
              <col name="el-table_1_column_4" width="180">
              <col name="el-table_1_column_5" width="180">
              <col name="el-table_1_column_6" width="180">
              <col name="el-table_1_column_7" width="180">
              <col name="el-table_1_column_8" width="180">
            </colgroup>
            <tbody>
              <tr class="el-table__row">
                <td rowspan="1" colspan="1" class="el-table_1_column_1  ">
                  <div class="cell">
                    <div class="el-input">
                      <!----><input type="text" autocomplete="off" placeholder="请输入姓名" class="el-input__inner">
                      <!---->
                      <!---->
                      <!---->
                      <!---->
                    </div>
                  </div>
                </td>
                <td rowspan="1" colspan="1" class="el-table_1_column_2   is-hidden">
                  <div class="cell">
                    <div class="el-input">
                      <!----><input type="text" autocomplete="off" placeholder="请输入账号" class="el-input__inner">
                      <!---->
                      <!---->
                      <!---->
                      <!---->
                    </div>
                  </div>
                </td>
                <td rowspan="1" colspan="1" class="el-table_1_column_3   is-hidden">
                  <div class="cell">
                    <div class="el-input el-input--suffix">
                      <!----><input type="password" autocomplete="off" placeholder="请输入密码" class="el-input__inner">
                      <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><!----><!----><i class="el-input__icon el-icon-view el-input__clear"></i><!----></span>
                      <!----></span>
                      <!---->
                      <!---->
                    </div>
                  </div>
                </td>
                <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center  is-hidden">
                  <div class="cell">
                    <div role="switch" aria-checked="true" class="el-switch is-checked"><input type="checkbox" name="" true-value="true" class="el-switch__input">
                      <!----><span class="el-switch__core" style="width: 40px; border-color: #13ce66; background-color: rgb(19, 206, 102);"></span>
                      <!---->
                    </div>
                  </div>
                </td>
                <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center  is-hidden">
                  <div class="cell">
                    <div role="switch" aria-checked="true" class="el-switch is-checked"><input type="checkbox" name="" true-value="true" class="el-switch__input">
                      <!----><span class="el-switch__core" style="width: 40px; border-color: #13ce66; background-color: rgb(19, 206, 102);"></span>
                      <!---->
                    </div>
                  </div>
                </td>
                <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center  is-hidden">
                  <div class="cell">
                    <div role="switch" aria-checked="true" class="el-switch is-checked"><input type="checkbox" name="" true-value="true" class="el-switch__input">
                      <!----><span class="el-switch__core" style="width: 40px; border-color: #13ce66; background-color: rgb(19, 206, 102);"></span>
                      <!---->
                    </div>
                  </div>
                </td>
                <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center  is-hidden">
                  <div class="cell">
                    <div role="switch" aria-checked="true" class="el-switch is-checked"><input type="checkbox" name="" true-value="true" class="el-switch__input">
                      <!----><span class="el-switch__core" style="width: 40px; border-color: #13ce66; background-color: rgb(19, 206, 102);"></span>
                      <!---->
                    </div>
                  </div>
                </td>
                <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center  is-hidden">
                  <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-round">
                      <!---->
                      <!----><span>提交</span></button> <button type="button" class="el-button el-button--plain el-button--small is-round">
                      <!---->
                      <!----><span>清空</span></button></div>
                </td>
              </tr>
              <!---->
            </tbody>
          </table>
          <!---->
        </div>
        <!---->
      </div>
      <!---->
      <!---->
      <div class="el-table__column-resize-proxy" style="display: none;"></div>
    </div>
  </main>
  <footer class="el-footer" style="height: 60px;">
    <div class="el-row is-justify-center el-row--flex"><button type="button" class="el-button el-button--primary el-button--medium is-round">
        <!---->
        <!----><span>一键提交</span></button> <button type="button" class="el-button el-button--plain el-button--medium is-round">
        <!---->
        <!----><span>一键清空</span></button></div>
    <div class="el-row is-justify-center el-row--flex"><button type="button" class="el-button el-button--primary el-button--medium is-round">
        <!---->
        <!----><span>继续添加</span></button></div>
  </footer>
</section>
`;
